test/acts_as_authentic_test/session_maintenance_test.rb

```diff
@@ -78,6 +78,33 @@ def test_updating_other_user
       assert_equal controller.cookies["user_credentials"], old_cookie_key
     end
 
+    def test_create_with_invalid_user
+      invalid_john = User.new(
+        :login => "invalid_johnny",
+        :password => "missing_stuff_dude",
+        :password_confirmation => "missing_stuff_dude",
+        :email => nil
+      )
+
+      assert_nothing_raised do
+        @invalid_user = UserSession.create(invalid_john)
+      end
```

Comment:
@cavi21 this assertion is probably unnecessary -- there's no expectation that it would raise an error now, after your last change.